OCP/Example1.md

```diff
@@ -0,0 +1,22 @@
+### Usage
+
+---
+#### Where
+https://github.com/keycloak/keycloak/blob/89795cfd7df24aad392c6223fe6d4639f03bd03d/core/src/main/java/org/keycloak/exceptions/TokenVerificationException.java#L27
```

Comment:
➕ ➖
What if we need to add more parameters in the future, should we add more overload methods? Is it really OCP?